README.md

```diff
@@ -1,59 +1,78 @@
 # Liberty Config Language Server
 
-Provides language server features for Liberty server configuration files. Supported files include 
-- `server.env`
-- `bootstrap.properties`
-- `server.xml` (or any XML file) and its referenced XML files through the `<include>` element
-    - XML file must contain the root element `<server>` and exist in `src/main/liberty/config`, `configDropins/overrides`, `configDropins/defaults`, `usr/shared/config`, or `usr/servers` directory
+The Liberty Config Language Server provides language server features for Liberty server configuration files through any of the supported client IDEs. It adheres to the [language server protocol](https://github.com/Microsoft/language-server-protocol).
+
+## Client IDEs
 
-Liberty Config Language Server adheres to the [language server protocol](https://github.com/Microsoft/language-server-protocol)
-and is available for use with the following clients.
+The Liberty Config Language Server is included in the following client IDEs.
 
-# Client IDEs
 * [Liberty Tools for VS Code](https://github.com/OpenLiberty/liberty-tools-vscode)
 * [Liberty Tools for Eclipse](https://github.com/OpenLiberty/liberty-tools-eclipse)
 * [Liberty Tools for Intellij](https://github.com/OpenLiberty/liberty-tools-intellij)
 
+## Supported files
+
+Liberty Config Language Server features are avaialble for the following configuration files.
+
+- `server.env`
+- `bootstrap.properties`
+- `server.xml` (or any XML file) and any XML files that are referenced through the `include` element in the `server.xml` file.
+    - XML files must contain the `server` root element and exist in the `src/main/liberty/config`, `configDropins/overrides`, `configDropins/defaults`, `usr/shared/config`, or `usr/servers` directory.
```

Comment:
Instead of making this a sub-bullet for the previous one, perhaps it should just be its own bullet:
- XML files that contain a `server` root element and are located in the ..., ..., ..., or ... directory.

Then change the previous bullet to:
- `server.xml` file and files referenced by an `include` element in the `server.xml` file.

Reply:
good idea- I'll make that change